src/aotcompile.cpp

```diff
@@ -1649,7 +1659,9 @@ void jl_dump_native_impl(void *native_code,
     if (z) {
         JL_TIMING(NATIVE_AOT, NATIVE_Sysimg);
         LLVMContext Context;
-        SetOpaquePointer(Context);
+        #if JL_LLVM_VERSION < 170000
+        SetOpaquePointer(ctx);
```

Comment:
Should this be
```suggestion
        SetOpaquePointer(Context);
```
?  Same below.